docs/rest_api/getting_started.rst

```diff
@@ -0,0 +1,99 @@
+Getting started
+###############
+
+Mautic provides several ``REST API`` endpoints that you can use. In the navigation menu on the left, you can find the available endpoints.
+
+.. warning:: 
+
+    By default, Mautic's ``REST API`` isn't enabled. The Mautic administrator can enable the API in the Mautic UI under ``Configuration -> API Settings``, or by setting ``'api_enabled' => 1`` in ``app/config/local.php`` directly.
+
+.. note:: 
+
+    Mautic has an API library available for PHP. You can `find it on GitHub <https://github.com/mautic/api-library>`_.
+
+To get started easily, you can use Mautic's Postman collection:
+
+.. image:: https://run.pstmn.io/button.svg
+   :target: https://app.getpostman.com/run-collection/19345380-9b7bbddc-8a4d-437a-8fc2-42b0b9823883?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D19345380-9b7bbddc-8a4d-437a-8fc2-42b0b9823883%26entityType%3Dcollection%26workspaceId%3D2c328b62-2531-4e35-a6bc-0f2995ce2df3
+
+Authentication
+**************
+
+Mautic supports Basic Authentication and OAuth2. Please see :doc:`authentication` for more details.
+
+Error handling
+**************
+
+In case of OAuth errors, the response is a JSON encoded array similar to:
+
+.. code-block:: json
+
+    {
+        "error": "invalid_grant",
+        "error_description": "The access token provided has expired."
+   }
+
+In case of system errors, the response is a JSON encoded array similar to:
+
+.. code-block:: json
+
+    {
+       "error": {
+           "message": "You do not have access to the requested area/action.",
+           "code": 403
+       }
+   }
+
+.. vale off
```

Comment:
Why are you turning off Vale here?

Reply:
Hi @RCheesley ...Thank you very much fo your feedback...it's my first time writing rst or hearing about vale....I pretty much copied the content that was linked in the PR you dropped, so i'm really not sure why it was initially added...hence, the reason i didn't take it off...but i'll look into it.